config/katello-devel.migrations/160503102719-gutterball.rb

```diff
@@ -0,0 +1,2 @@
+answers.delete('katello_devel::plugin::gutterball') if answers['katello_devel::plugin::gutterball']
+answers.delete('katello_devel::plugin::foreman_gutterball') if answers['katello_devel::plugin::foreman_gutterball']
```

Comment:
If for some reason these were set to false, I don't think this would actually remove them.